features/navbar.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Visitor Can Navigate via Navbar
+    As a visitor,
+    In order to navigate to different sections of the site,
+    I would like to be able to use a navbar
+
+  Scenario: Visitor Sees Navbar Links on Main Page
+      Given I visit the site
+      Then I should see "HOME"
+      Then I should see "LOGIN"
```

Comment:
Please refactor to 
`And I should see 'Login'`

etc etc

Reply:
Done